PlayerOne/index.js

```diff
@@ -46,17 +54,33 @@ $(document).ready(function() {
 })
 
 //  web socket connection
+// route address
 const url = "wss://i788c.sse.codesandbox.io/";
-const guessConnection = new WebSocket(url + "chosen-word"); // create a new websocket connection to port 8080 - connect to specific endpoint 'submit-word'
+
+// connection to send chosen word to others
+const chosenWordConnection = new WebSocket(url + "chosen-word"); // create a new websocket connection to port 8080 - connect to specific endpoint 'submit-word'
```

Comment:
this part was quite messy and I accidentally mixed up the different connections for the given endpoint